datachecks/core/datasource/sql_datasource.py

```diff
@@ -143,6 +143,22 @@ def query_get_null_count(self, table: str, field: str, filters: str = None) -> i
             query += " AND {}".format(filters)
         return self.connection.execute(text(query)).fetchone()[0]
 
+    def query_get_empty_string_count(
+        self, table: str, field: str, filters: str = None
+    ) -> int:
+        """
+        Get the count of empty strings in a column of a table
+        :param table: table name
+        :param field: column name
+        :param filters: filter condition
+        :return: count of empty strings
+        """
+        query = "SELECT COUNT(*) FROM {} WHERE {} = ''".format(table, field)
```

Comment:
the table name needs to be changed.

Use `qualified_table_name` in place of table.

```python
qualified_table_name = self.qualified_table_name(table)
query = "SELECT MAX({}) FROM {}".format(field, qualified_table_name)
```

Reply:
Done, It's added!